Bottom: f58aa69c3adee5cef59558b360220f1b21d5fdfb
Top:    ed21fa4a98215d60605119e7a3faa288bd9169cb
Author: Christopher Voltz <cjunk@voltz.ws>
Date:   2016-04-29 01:43:43 -0500

Update README and convert to English


---

diff --git a/README.md b/README.md
index e3ade3f..6238caa 100644
--- a/README.md
+++ b/README.md
@@ -1,7 +1,12 @@
 tablecalc
 =========
 
-dokuwiki plugin nejen pro FO. Časem bude patrně nahrazen.
+Dokuwiki plugin which adds limited spreadsheet functionality to tables. See
+the original documentation page at: https://www.dokuwiki.org/plugin:tablecalc.
 
-Fork: https://www.dokuwiki.org/plugin:tablecalc
+Forked from https://github.com/pirati-cz/tablecalc. Adds the following
+functions:
 
+  | ''tag(name)''     | Associate the current row and column with a name |
+  | ''tag_col(name)'' | Return the column for the specified tag          |
+  | ''tag_row(name)'' | Return the row for the specified tag             |
